cuqi/testproblem/_testproblem.py

```diff
@@ -148,69 +152,109 @@ class Deconvolution1D(BayesianProblem):
+        | 'Moffat' - a 1D Moffat blur function
+        | 'Defocus' - an out-of-focus 1D blur function
+        | 'sinc' or 'prolate' - a sinc function (LEGACY only)
+        | 'vonMises' - a periodic version of the Gauss function (LEGACY only)
+        | ndarray - a custom PSF represented as a 1D ndarray.
 
-    kernel_param : scalar
-        A parameter that determines the shape of the kernel;
-        the larger the parameter, the slower the initial
-        decay of the singular values of A.
-        Ignored if kernel is a ndarray.
+    PSF_param : scalar
+        | A parameter that determines the shape of the PSF;
+        | the larger the parameter, the larger the blur on the signal.
+        | Ignored if PSF is a ndarray.
+
+    PSF_size : int
+        | The size of the PSF.
+        | Ignored if PSF is a ndarray.
+
+    BC : string
+        | Boundary conditions for the convolution.
+        | 'zero' - zero boundary conditions
+        | 'periodic' - periodic boundary conditions
+        | 'Neumann' - Neumann boundary conditions
+        | 'Mirror' - Mirrored boundary conditions
+        | 'Nearest' - Replicates last element of boundary
 
     phantom : string or ndarray
-        The phantom that is sampled to produce x
-        'Gauss' - a Gaussian function
-        'sinc' - a sinc function
-        'vonMises' - a periodic version of the Gauss function
-        'square' - a "top hat" function
-        'hat' - a triangular hat function
-        'bumps' - two bumps
-        'derivGauss' - the first derivative of Gauss function
-        'pc' - Piece-wise constant phantom
-        ndarray - a custom phantom
+        | The phantom that is sampled to produce the exact solution (signal).
+        | 'Gauss' - a Gaussian function
+        | 'sinc' - a sinc function
+        | 'vonMises' - a periodic version of the Gauss function
+        | 'square' - a "top hat" function
+        | 'hat' - a triangular hat function
+        | 'bumps' - two bumps
+        | 'derivGauss' - the first derivative of Gauss function
+        | 'pc' - Piece-wise constant phantom
+        | ndarray - a custom phantom
 
     phantom_param : scalar
-        A parameter that determines the width of the central 
-        "bump" of the function; the larger the parameter,
-        the narrower the "bump."  
-        Does not apply to phantom = 'bumps' or ndarray.
+        | A parameter that determines the width of the central 
+        | peak of the function; the larger the parameter,
+        | the narrower the peak.  
+        | Does not apply to phantom = 'bumps' or ndarray.
 
     noise_type : string
-        The type of noise
-        "Gaussian" - Gaussian white noise¨
-        "scaledGaussian" - Scaled (by data) Gaussian noise
+        | The type of noise
+        | "Gaussian" - Gaussian white noise¨
+        | "scaledGaussian" - Scaled (by data) Gaussian noise. Here noise is scaled
+        | by each data point individually.
```

Comment:
I appreciate trying to stick to the brief format here, yet it is in my eyes not completely clear what scaled means. Could you please expand either here or later in the documentation to specify precisely how this is scaled.